### DISCUSSION TITLE
In-vehicle Network - Showcase

### DISCUSSION THREAD

**Discussion by adamgeorge309** [inet-framework/inet#785] [Showcases]
Discuss the "In-vehicle Network - Showcase" showcase here. Ideas and recommendations are welcome.

**Comment by tuantitu22** [comment]
Hello,
When I run the demo, I found that UDP multicast is OK for standard ethernet. For example, frontRightCamera.app[0] sends out 81 packets, then hud.app[3] and obu.app[3] receive 80 packets. However, when it is TSN network, frontRightCamera.app[0] works fine, but hud.app[3] and obu.app[3] cannot receive anything. Could you please tell me how to fix it?

**Reply by tuantitu22** [reply]
I have done another test. The same topology, when I set the destination node to standardHost, it can receive multicast packages. But when change it into TsnDevice, it cannot. Because TsnDevice is configured as EthernetLayer. But still, I don't know how to fix it.
